### DISCUSSION TITLE
Creating a Pull Request - LGMSOC Participant(Keerthana)

### DISCUSSION THREAD

**Discussion by prathimacode-hub** [prathimacode-hub/ML-ProjectKart#103] [Q&A]
I shall share you the easiest way to create a pull request without any hassles especially for beginners to open source and github

**Comment by prathimacode-hub** [comment]
Process to Follow (without command line confusion)👍

Fork the repository. You are now in your profile repository.
You can see a "Add File" option in the local repository
You can add files or folders using the 2options given
Create new file option helps in creating both file and sub repository within this repository.
Example : You want to add up your "Spam Email Detection" Script to the repository
Go to "ML-ProjectKart" repository. Select "Create a new file" option,
It opens up like this - /ML-ProjectKart
To add up your repository and file you do :
/ML-ProjectKart/Spam Email Detection/README.md
This add up would not only add 'Spam Email Detection' as repository but also adds a empty 'README.md' file.
Now you have created a repository of your desired script, you are yet to add your project related files.
Now we would create sub folders inside "Spam Email Detection". There would be 2 repositories inside.
Let's create it. To add up your repository and file you do :
/ML-ProjectKart/Spam Email Detection/Model/README.md
You can add your README content in 'README.md file' and commit the changes
From the very same "Add File" option, select "Upload files" and add your .ipynb file from your local drive of your system.
Make sure the script file name is "spam_email_detection.ipynb" before you upload the file to this repository.
Now, we would create another folder in "Spam Email Detection". To add up your repository and file you do :
/ML-ProjectKart/Spam Email Detection/Dataset/README.md
Add the dataset files of your project (.csv files) into "Dataset" folder.
If we remember, we had created two extra 'README.md' files in "Dataset" folder and "Spam Email Detection" as well to get it registered as a repository.
You can now delete that files  separately by opening that files and using "delete option that is trash button". And you are done.
Finally, You have successfully added up all your required files needed.
Now select "Contribute" option above and select Open Pull request, and add a defined title of your project, a proper commit message in a description using the PR template given.
Finally, "Create a Pull Request" option to synchronize your fork to the committed repository. That's it.
Voila, you did it. Congrats, on learning how to create a PR in a easier way. No much fuss around to deal with.

PR can be created without the use of command line terminal, but preferrably that way to show professionalism.
Contribution is all that matters, so start contributing and feel free to do more PR in future.

@, I like your confidence and enthusiasm to learn. Keep going.
If you found this helpful and easier to try, you can do create the PR of the same script with required changes I have mentioned in this comment.

Looking forward to merge the upcoming PR from you. Thank You.

**Reply by sai-keerthi** [reply]
Thank you so much for your support mam
